sklearn/datasets/_samples_generator.py

```diff
@@ -1828,17 +1827,15 @@ def make_checkerboard(
     cols : ndarray of shape (n_clusters, X.shape[1])
         The indicators for cluster membership of each column.
 
+    See Also
+    --------
+    make_biclusters : Generate an array with constant block diagonal structure for biclustering.
```

Comment:
There is a [linting error](https://dev.azure.com/scikit-learn/scikit-learn/_build/results?buildId=37733&view=logs&jobId=32e2e1bb-a28f-5b18-6cfc-3f01273f5609) on the CI.

```suggestion
    make_biclusters : Generate an array with constant block diagonal structure
        for biclustering.
```

Reply:
sure sir